iset.mm

```diff
@@ -182512,6 +182550,350 @@ mathematician Ptolemy (Claudius Ptolemaeus).  This particular version is
+
+    $( The value of the divisor function.  (Contributed by Mario Carneiro,
+       21-Jun-2015.) $)
+    sgmval2 $p |- ( ( A e. ZZ /\ B e. NN ) ->
+      ( A sigma B ) = sum_ k e. { p e. NN | p || B } ( k ^ A ) ) $=
+      ( cz wcel cn wa csgm co cv cdvds wbr crab ccxp csu cexp cc wceq zcn sylan
+      sgmval ssrab2 simpr sselid simpll cxpexpnn syl2anc sumeq2dv eqtrd ) AEFZB
+      GFZHZABIJZDKBLMZDGNZCKZAOJZCPZUPUQAQJZCPUKARFULUNUSSATABCDUBUAUMUPURUTCUM
+      UQUPFZHZUQGFUKURUTSVBUPGUQUODGUCUMVAUDUEUKULVAUFUQAUGUHUIUJ $.
+
+    $( The value of the sum-of-divisors function, usually denoted
+       &sigma;<SUB>0</SUB>(<i>n</i>).  (Contributed by Mario Carneiro,
+       21-Jun-2015.) $)
+    0sgm $p |- ( A e. NN -> ( 0 sigma A ) = ( # ` { p e. NN | p || A } ) ) $=
+      ( vk cn wcel cc0 csgm co cv cdvds wbr crab cexp csu chash c1 cmul cz wceq
+      cfv 0z sgmval2 mpan elrabi nncnd exp0d sumeq2i cc dvdsfi ax-1cn fsumconst
+      cfn sylancl eqtrid cn0 hashcl syl nn0cnd mulridd 3eqtrd ) ADEZFAGHZBIAJKZ
+      BDLZCIZFMHZCNZVDOTZPQHZVHFREVAVBVGSUAFACBUBUCVAVGVDPCNZVIVDVFPCVEVDEZVEVK
+      VEVCBVEDUDUEUFUGVAVDULEZPUHEVJVISBAUIZUJVDPCUKUMUNVAVHVAVHVAVLVHUOEVMVDUP
+      UQURUSUT $.
+
+    $( The divisor function is a function into the complex numbers.
+       (Contributed by Mario Carneiro, 22-Sep-2014.)  (Revised by Mario
+       Carneiro, 21-Jun-2015.) $)
+    sgmf $p |- sigma : ( CC X. NN ) --> CC $=
+      ( vp vn vk vx cv cdvds wbr cn crab ccxp co csu cc wcel wral cxp wf adantl
+      csgm wa cfn dvdsfi crp elrabi nnrpd simpll rpcncxpcld fsumcl rgen2 df-sgm
+      fmpo mpbi ) AEBEZFGZAHIZCEZDEZJKZCLZMNZBHODMOMHPMSQUTDBMHUQMNZUMHNZTZUOUR
+      CVBUOUANVAAUMUBRVCUPUONZTUPUQVDUPUCNVCVDUPUNAUPHUDUERVAVBVDUFUGUHUIDBMHUS
+      MSDCBAUJUKUL $.
+
+    $( Closure of the divisor function.  (Contributed by Mario Carneiro,
+       22-Sep-2014.) $)
+    sgmcl $p |- ( ( A e. CC /\ B e. NN ) -> ( A sigma B ) e. CC ) $=
+      ( cc cn csgm sgmf fovcl ) ABCCDEFG $.
+
+    $( Closure of the divisor function.  (Contributed by Mario Carneiro,
+       21-Jun-2015.) $)
+    sgmnncl $p |- ( ( A e. NN0 /\ B e. NN ) -> ( A sigma B ) e. NN ) $=
+      ( vp vk cn0 wcel cn wa csgm co cv cdvds wbr crab cexp csu wceq adantl syl
+      cz nn0z sgmval2 sylan cc0 clt dvdsfi elrabi simpl nnexpcl syl2anr fsumzcl
+      cfn nnzd c0 wne wrex nnz iddvds breq1 rspcev mpdan rabn0r fsumrpcl rpgt0d
+      nnrpd elnnz sylanbrc eqeltrd ) AEFZBGFZHZABIJZCKZBLMZCGNZDKZAOJZDPZGVIATF
+      VJVLVRQAUAABDCUBUCVKVRTFUDVRUEMVRGFVKVOVQDVJVOULFVICBUFRZVKVPVOFZHZVQVTVP
+      GFVIVQGFVKVNCVPGUGVIVJUHVPAUIUJZUMUKVKVRVKVOVQDVSVJVOUNUOZVIVJVNCGUPZWCVJ
+      BBLMZWDVJBTFWEBUQBURSVNWECBGVMBBLUSUTVAVNCGVBSRWAVQWBVEVCVDVRVFVGVH $.
+  $}
+
+  ${
+    $d m n x A $.  $d m n x P $.
+    dvdsppwf1o.f $e |- F = ( n e. ( 0 ... A ) |-> ( P ^ n ) ) $.
+    $( A bijection from the divisors of a prime power to the integers less than
+       the prime count.  (Contributed by Mario Carneiro, 5-May-2016.) $)
```

Comment:
What is the meaning of "prime count" here? The "the divisors of a prime power" is the range of F, and the domain is the number of exponents.